### DISCUSSION TITLE
CyberPet：基于Walrus的宠物NFT铸造项目

### DISCUSSION THREAD

**Discussion by pixiudecp** [creatorsand/.github#125] [Ideas]
##代码链接
https://github.com/pixiudecp/cyberpet.git

## 一句话 IDEA：
CyberPet：为自己的爱宠铸造NFT，并将媒体文件储存到Walrus上，实现完全自己拥有的线上爱宠。

## 我的角色：
项目发起人+产品经理

## 社群昵称：
Michael，ChatGPT

## 个人简介：
跨行业连续创业者

## 产品是给谁用的？
目标客户群：有宠物的web3用户
需求：将自己宠物铸造成NFT 永久存于链上，就像是元宇宙中的宠物副本。并且可以传承，可以交易。

## 其他产品有什么痛点，用户为什么要选择你的产品？

传统nft将媒体文件存于中心化服务器， 文件所有权无法掌控，且有服务器宕机风险。
而将图片存于IPFS不仅速度缓慢，而且也无法得到永久可用保证，如果gateway失效或不pin这个文件，都会造成不可用。

我们通过将宠物NFT的媒体文件部署在Walrus 实现NFT的永久化保证，使用户的爱宠NFT的所有权真正掌控在自己手中。

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]
